src/tab.js

```diff
@@ -133,15 +133,19 @@ SideTab.prototype = {
     toggleClass(this._iconOverlayView, "muted", muted);
   },
   updateIcon(favIconUrl) {
```

Comment:
Instead of changing that method, how about we call resetIcon directly here: https://github.com/eoger/tabcenter-redux/blob/017d29e768ef0cba64da70fc14b8feea8378b2b6/src/tablist.js#L648-L653